main/webapp/modules/core/scripts/index/import-project-ui.html

```diff
@@ -3,15 +3,15 @@
     <tr>
       <td colspan="2" id="or-import-locate"></td>
     <tr>
-      <td id="or-import-file"></th>
-      <td><input type="file" id="project-tar-file-input" name="project-file" /></td>
+      <td id="or-import-file"></td>
+      <td><input type="file" id="project-tar-file-input" name="project-file"  value="" /></td>
     </tr>
     <tr>
       <td id="or-import-rename"></th>
       <td><input type="text" size="25" id="project-name-input" name="project-name" /></td></tr>
     <tr>
       <td></td>
-      <td><input type="submit" id="import-project-button" class="button button-primary" /></td>
+      <td><input type="button"  id="import-project-button" class="button button-primary" onclick="inputChecker()"/></td>
```

Comment:
It would better not to bind `onclick` here, but rather do it from the JS file `import-project-ui.js`. I suggest you modify the existing event listener on this button to perform the check there.